homeassistant/components/smart_meter_b_route/config_flow.py

```diff
@@ -0,0 +1,75 @@
+"""Config flow for Smart Meter B Route integration."""
+
+import logging
+from typing import Any
+
+from momonga import Momonga, MomongaSkJoinFailure, MomongaSkScanFailure
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_DEVICE, CONF_ID, CONF_PASSWORD
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN, ENTRY_TITLE
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_DEVICE): str,
+        vol.Required(CONF_ID): str,
```

Comment:
But what is a device and what is an id? I am assuming id is an username and that we use the device to specify the device. Is it possible to set up all devices we find on the account?

Reply:
The device is the `/dev/tty*` device used to establish the serial connection.

The id is similar to a username but is refered as id in the documentation found on internet

Reply:
Checkout `energenie_power_sockets` that's also an USB based device, maybe that can help

Reply:
Hum... It relies too much on the associated API...

I found some useful code in ZHA but I need a deep dive to see how to separate from the API.